packages/pages/src/dev/server/autoInit.ts

```diff
@@ -26,7 +26,7 @@ const autoYextInitWithYextrc = async (scope: string | undefined) => {
   }
 };
 
-const createYextrc = async (scope: string | undefined) => {
+const createYextrc = () => {
   fs.writeFileSync(".yextrc", "");
```

Comment:
This function just calls another function and only in one place. Call fs.writeFileSync directly on line 10.